src/mesh/RF95Interface.cpp

```
@@ -212,6 +212,8 @@ bool RF95Interface::reconfigure()
 
     // set mode to standby
     setStandby();
+    if (disabled)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)